packages/core/admin/admin/src/components/OverlayBlocker/Overlay.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+const Overlay = styled.div`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1140;
```

Comment:
Do you know how this z-index value has been defined?

Reply:
It was like this in the helper-plugin I thought it was easier to just copy the file for now and improve it when we change its design.